live-examples/css-examples/basic-user-interface/appearance.html

```diff
@@ -0,0 +1,39 @@
+<section id="example-choice-list" class="example-choice-list">
+  <div class="example-choice">
+    <pre><code class="language-css">-moz-appearance: none;
+appearance: none;</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Copy to Clipboard</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">-moz-appearance: button;
+apperance: button;</code></pre>
```

Comment:
apperance -> appearance, here and in the next two occurrences.

But wouldn't it be better to use `-webkit-appearance` throughout? Then it would also work in Chrome.

Reply:
OK, yeah. In place of the non-functioning `appearance` (and typo variant), I've put in the `-webkit-` prefixed versions.